### PR TITLE
[IT-3385] Bumps tower_version to Patched version

### PR DESCRIPTION
**Pull request by BWMac**
**Problem:**

When running workflows with a very large number of processes in Tower, workflows will fail.

**Solution:**

We have been in contact with Seqera about this issue for ~3 months now. They have finally published a patched version of the frontend container image we use to provide a fix. So, this PR bumps the `tower_version` we are using to this patched version.

I will post an announcement in the `tower-news` slack channel and give users a chance to see that this change is coming before merging.

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -1,20 +1,21 @@
 project_code: aws-workflows-nextflow-infra
-profile: {{ var.profile | default() }}
-region: {{ var.region | default("us-east-1") }}
+profile: {{var.profile | default()}}
```
Comment (zaro0508) [inline config/config.yaml]: any reason for all these formatting changes?

**Review by zaro0508 — APPROVED** [review]
thanks for the PR.

**Review by zaro0508 — CHANGES_REQUESTED** [review]
ohh wait, this PR should be based on `dev` branch not `prod` branch

File: config/config.yaml (hunk 1 above)
Comment (BWMac) [reply config/config.yaml]: My auto-formatter got me, reverted in the latest commit.

**Comment by thomasyu888** [comment]
@zaro0508 since the dev branch is currently in the process of being updated to the newest version, we are doing a minor patch release to prod so we can move our work along.

**Review by thomasyu888 — APPROVED** [review]

**Comment by zaro0508** [comment]
i think the better thing to do is patch dev -> patch prod -> move dev to latest

**Comment by thomasyu888** [comment]
@BWMac Lets follow @zaro0508 's suggestion, hopefully that doesn't break dev to move backwards.

**Comment by zaro0508** [comment]
this PR looks wrong

**Comment by BWMac** [comment]
This PR was weird because my new branch was originally based on `prod` and then I changed the base to `dev`. There were changes in #267  and #272 which were pushed straight into `prod` and have now been merged into `dev` in this PR. `tower-dev` does appear to be working after the re-deploy, however.

**Pull request merged**